systemd-udevd/pkg.yaml

```diff
@@ -0,0 +1,114 @@
+        patch -p1 < /pkg/patches/0026-undef-stdin-for-references-using-stdin-as-a-struct-m.patch
+        patch -p1 < /pkg/patches/0027-adjust-header-inclusion-order-to-avoid-redeclaration.patch
+        patch -p1 < /pkg/patches/0028-build-path.c-avoid-boot-time-segfault-for-musl.patch
+
+        # Own patches
+        patch -p1 < /pkg/patches/0001-chore-musl-headers-compatibility-for-udev.patch
+
+        # for full build add this:
+        #-Dnss-mymachines=disabled -Dnss-myhostname=false -Dnss-resolve=disabled -Dnss-systemd=false
+        meson setup build \
+          --buildtype=release \
+          -Dmode=release \
+          -Dversion-tag="{{ .systemd_version }}-talos" \
+          -Dselinux=enabled \
+          -Dblkid=enabled \
+          -Dkmod=enabled \
+          -Dopenssl=disabled \
+          -Dgshadow=false \
+          -Dutmp=false \
+          -Duserdb=false \
+          -Ddbus=disabled \
+          -Dglib=disabled \
+          -Dlink-udev-shared=false \
+          -Dman=disabled \
+          -Dtests=false \
+          -Dwerror=false
+    build:
+      - |
+        # rules.d/64-btrfs.rules is disabled in our build, other are static
+        # Buildable rules from rules.d/meson.build
+        ninja -j $(nproc) -C build udevadm rules.d/50-udev-default.rules rules.d/71-seat.rules rules.d/90-vconsole.rules \
+          rules.d/60-persistent-storage.rules rules.d/70-uaccess.rules rules.d/73-seat-late.rules rules.d/99-systemd.rules \
+          ata_id cdrom_id dmi_memory_id fido_id iocost mtd_probe scsi_id v4l_id \
+          libudev.so.1.7.9 libudev.a src/libudev/libudev.pc src/udev/udev.pc
+        # Build all instead and copy necessary parts?
+        # ninja -j $(nproc) -C build
+    install:
+      - |
+        mkdir -p /rootfs/usr/bin
+        cp build/udevadm /rootfs/usr/bin/
+        mkdir -p /rootfs/sbin
+        ln -s /rootfs/usr/bin/udevadm /rootfs/sbin/udevadm
+        ln -s /rootfs/usr/bin/udevadm /rootfs/sbin/systemd-udevd
+
+        mkdir -p /rootfs/usr/lib/udev/rules.d/
+        cp build/rules.d/*.rules /rootfs/usr/lib/udev/rules.d/
+        cp rules.d/*.rules /rootfs/usr/lib/udev/rules.d/
+        rm -f /rootfs/usr/lib/udev/rules.d/{60-cdrom_id.rules,60-persistent-alsa.rules,60-persistent-v4l.rules,64-btrfs.rules,70-joystick.rules,70-mouse.rules,70-touchpad.rules,78-sound-card.rules,80-net-name-slot.rules}
+        # Azure csi support
+        cp /pkg/files/66-azure.rules /rootfs/usr/lib/udev/rules.d/66-azure.rules
+
+        # From src/udev/meson.build, used by rules
+        cp build/{ata_id,cdrom_id,dmi_memory_id,fido_id,iocost,mtd_probe,scsi_id,v4l_id} /rootfs/usr/lib/udev/
+
+        mkdir -p /rootfs/usr/lib/pkgconfig
+        mkdir -p /rootfs/usr/share/pkgconfig
+        cp build/src/libudev/libudev.pc /rootfs/usr/lib/pkgconfig/
+        cp build/src/udev/udev.pc /rootfs/usr/share/pkgconfig/
+
+        cp build/libudev.{a,so{.1.7.9,.1,}} /rootfs/usr/lib/
```

Comment:
Perhaps we should create `libudev.so.1` and `libudev.so` as symlinks here. Will fix after major stuff